oak_functions/examples/weather_lookup/module/src/tests.rs

```diff
@@ -158,3 +162,55 @@ fn test_location_from_slice() {
     assert_eq!(bytes, value.to_bytes(),);
     assert_eq!(value, Location::from_bytes(&bytes));
 }
+
+// This benchmark test takes a very long time, so is not run during tests.
+// To actually run the bench, use `cargo bench --features run-internal-bench`.
+#[cfg_attr(feature = "run-internal-bench", bench)]
+#[cfg_attr(not(feature = "run-internal-bench"), allow(dead_code))]
+fn bench_wasm_handler(bencher: &mut Bencher) {
+    let mut manifest_path = std::env::current_dir().unwrap();
+    manifest_path.push("Cargo.toml");
+    let wasm_module_bytes =
+        test_utils::compile_rust_wasm(manifest_path.to_str().expect("Invalid target dir"))
+            .expect("Couldn't read Wasm module");
+    let mut rng = rand::thread_rng();
+    let buf = generate_and_serialize_sparse_weather_entries(&mut rng, 10_000).unwrap();
```

Comment:
Could the size be parametric based on the feature (`run-internal-bench`) and included in the CI with a smaller size so that the test won't stale?

Reply:
Good idea. Done